### ISSUE TITLE
show number of tasks to be modified

### ISSUE THREAD

**Issue by jaker-dotcom** [GothenburgBitFactory/taskwarrior#2532]
When entering a modification that is not bound to one or multiple tasks by their ID but by a filter (e.g. project and pending) one is asked whether all tasks that the criteria applies to should be modified. Otherwise one can confirm or decline per task. I find myself oftentimes going "yes .. yes .. yes .." because there's no way to tell whether the filter was correct (as in I didn't do a mistake there). If it showed the number of all tasks that the  criteria applies to it would be an indicator for whether it's right or wrong. Drawback from having a filter that is too broad (say forgot to add +PENDING) is that you have to run the undo command several times.

**Comment by tbabej** [comment]
This is a great suggestion and relatively self-contained. Putting into our next milestone.

**Comment by sebu06** [comment]
I had some code that also listed the number of changes for the tasks:
```
➜  build git:(2.6.0) ✗ ./src/task pro:A done
This command will alter 3 tasks.
  - End will be set to '2021-07-18'. (3 tasks)
  - Start will be deleted. (1 tasks)
  - Status will be changed from 'pending' to 'completed'. (3 tasks)
```

However, for this I changed the return value of taskDifferences to a std::vector<std::string> and had to call taskDifferences twice. In the end I didn't like this approach and reverted it. However, if that is a wanted feature, I could push this as well.

**Comment by sebu06** [comment]
implemented with PR #2538

**Comment by jaker-dotcom** [comment]
Great thanks!
